examples/multi-agent-debate/config.py

```diff
@@ -0,0 +1,70 @@
+from rich.prompt import Prompt, Confirm
+import langroid.language_models as lm
+import langroid.utils.configuration
+
+def get_global_settings(debug=False, nocache=False):
+    """
+    Returns global settings for Langroid.
+
+    Args:
+        debug (bool): Enable or disable debug mode.
+        nocache (bool): Enable or disable caching.
+
+    Returns:
+        Settings: Langroid configuration settings.
+    """
+    return langroid.utils.configuration.Settings(
+        debug=debug,
+        cache=not nocache,
+    )
+
+def get_base_llm_config():
+    """
+    Prompts the user to select a base LLM configuration.
+
+    Returns:
+        OpenAIGPTConfig: Base configuration for the selected LLM.
+    """
+    chat_model_option = Prompt.ask(
+        "Which OpenAI Model do you want to use? Select an option:\n"
+        "1: GPT4o\n"
+        "2: GPT4\n"
+        "3: Mistral: mistral:7b-instruct-v0.2-q8_0a\n"
+        "Enter 1, 2, or 3:",
+        choices=["1", "2", "3"],
+        default="1"
+    )
+
+    model_map = {
+        "1": "GPT4o",
```

Comment:
use the actual openai mdl names here , e.g. `gpt-4`, `gpt-4o`, then directly use it to set `chat_model`, no need for `getattr`